### PR TITLE
style(success messages): deintensify the colour

### LINKED ISSUE #6943: Deintensify the colour of our success messages
Current:
image

Proposed:
image

New background colour: #c7fbbf
New text and link colour: #063311

Also checks out for contrast ratio.
https://contrast-ratio.com/#%23063311-on-%23c7fbbf

Still green but less intense…
giphy

### PR DESCRIPTION
**Pull request by shreya99oak**
fixes #6943 

@shane-tomlinson Pls. review

### CODE REVIEW

**Comment by shreya99oak** [comment]
Ohk, I'll keep that in mind!
Should I create a new branch and resubmit a PR?
 I'll see why the css wasn't affected. Do I have to change npm-shrinkwrap.json  too?

**Comment by shane-tomlinson** [comment]
> Should I create a new branch and resubmit a PR?

It should be OK for this one, it's more for your local housekeeping.

> I'll see why the css wasn't affected. Do I have to change npm-shrinkwrap.json too?

npm-shrinkwrap.json should only update if you install new dependencies. Instead of installing dependencies using `npm install`, we recommend using `npm ci`.

**Comment by shreya99oak** [comment]
@shane-tomlinson Pls. review it

File: node_modules/photon-colors/colors.scss [hunk 1]
```diff
@@ -0,0 +1,65 @@
+/* Firefox Colors SCSS Variables v1.0.3 */
```
Comment (shane-tomlinson) [inline node_modules/photon-colors/colors.scss]: Files in node_modules should not be checked in, both `npm install` and `npm ci` would clobber your changes.

Instead, change the colors here: https://github.com/shreya99oak/fxa-content-server/blob/1acf910ec4b0ba8c54f4e7a5356b76a4debb2fb7/app/styles/_variables.scss#L131:L132

**Comment by miralobontiu** [comment]
Verified as fixed on Stage - Train 132.

**Pull request merged**